core/src/main/java/io/methvin/watcher/DirectoryWatcher.java

```diff
@@ -346,7 +346,15 @@ private void notifyCreateEvent(Path path, int count) throws IOException {
     if (fileHasher != null || Files.isDirectory(path)) {
       HashCode newHash = PathUtils.hash(fileHasher, path);
       if (newHash == null) {
-        logger.debug("Failed to hash created file [{}]. It may have been deleted.", path);
+        // Hashing could fail for locked files on Windows
+        // Skip notification only if we confirm the file does not exist
+        if (Files.notExists(path)) {
+          logger.debug("Failed to hash created file [{}]. It may have been deleted.", path);
+        } else {
+          logger.debug("Failed to hash created file [{}]. It may be locked.", path);
+          logger.debug("{} [{}]", EventType.CREATE, path);
+          listener.onEvent(new DirectoryChangeEvent(EventType.CREATE, path, count));
+        }
         return;
       }
```

Comment:
Now that this the above case has gotten more complicated, I would change the below to an `else if` and remove the `return`. I think that's slightly clearer.

Reply:
PR updated